src/ensembl/src/content/app/entity-viewer/components/entity-viewer-topbar/scale-switcher/ScaleSwitcher.scss

```diff
@@ -1,3 +1,25 @@
+@import 'src/styles/common';
+
 .container {
-  /* css to come later */
+  display: flex;
+}
+
+.selectedNode {
+  border-radius: 12px;
+  height: 24px;
+  padding: 1px 20px 0 20px;
+  margin-right: 24px;
+  background-color: $black;
+  color: $white;
+  text-align: center;
+  cursor: default;
+}
+
+.emptyNode {
+  border-radius: 10px;
+  background-color: $grey;
+  margin: 6px 24px 0 0;
+  display: block;
```

Comment:
Empty node is a div, right? So it will have display: block by default, which makes this rule redundant.